lib/widgets/sheets/draggable_scrollable.dart

```diff
@@ -45,9 +45,7 @@ class DraggableScrollable extends StatelessWidget {
         decoration: BoxDecoration(
           color: Color(0xffefeef3),
         ),
-        child: SingleChildScrollView(
```

Comment:
I don't think we should declare `SingleChildScrollView` here, it makes child widgets easily get bugs with scroll if they have a `ListView` inside it. So I suggest we declare `SingleChildScrollView` deeper in child widgets only when we need.